datadog/dogshell/downtime.py

```diff
@@ -44,15 +44,14 @@ def setup_parser(cls, subparsers):
         show_parser.add_argument('downtime_id', help="downtime to show")
         show_parser.set_defaults(func=cls._show_downtime)
 
-        show_all_parser = verb_parsers.add_parser('show_all', help="Show a list of all downtimes")
-        show_all_parser.add_argument('--current_only', help="only return downtimes that"
-                                     " are active when the request is made", default=None)
-        show_all_parser.set_defaults(func=cls._show_all_downtime)
-
```

Comment:
Are these not needed anymore?

Reply:
The dogshell `downtime` command is currently broken because we don't support the `show_all` downtimes using the client. I removed this to implement the `cancel/by_scope` endpoint. I can fix the `show_all` command, but could I do that in a separate PR?

Reply:
Sure let's fix that in a separate PR, but why remove it at all in this one ? Was it preventing you from doing anything ?

Reply:
Yeah, the dogshell `dog downtime` command won't run currently.

Reply:
Hmm I just tried from master the `show_all` command, and it works. Maybe something wrong with your setup ?